test/test_jit.py

```diff
@@ -5058,6 +5058,17 @@ def forward(self):
         traced = torch.jit.trace(TryTracing123(), ())
         self.assertEqual(torch.zeros(4, 4), traced())
 
+    @skipIfRocm
+    @unittest.skipIf(IS_WINDOWS, "TODO: Fix this test case")
+    def test_torchbind_pickle_serialization(self):
```

Comment:
Can you also test that it is loadable as an attribute? Just add it to this list and run `TestScript.test_attribute_serialization`

https://github.com/pytorch/pytorch/blob/040bc1d0e1eac9c50072332fc4d939a90823099c/test/test_jit.py#L16513